lib/room.rb

```diff
@@ -1,12 +1,22 @@
-class Rom
+class Room
+    attr_reader :category, :length, :width
```

Comment:
Since we don't ask for `length` or `width` methods, these don't need to be attr_readers.